apps/native-component-list/src/screens/GL/GLMaskScreen.tsx

```diff
@@ -102,11 +99,7 @@ export default class GLMaskScreen extends React.Component<Props> {
         // Buffer data and draw!
         const speed = this.props.speed || 1;
         const a = 0.48 * Math.sin(0.001 * speed * Date.now()) + 0.5;
-        const verts = new Float32Array([
-          -a, -a, a, -a,
-          -a, a, -a, a,
-          a, -a, a, a,
-        ]);
+        const verts = new Float32Array([-a, -a, a, -a, -a, a, -a, a, a, -a, a, a]);
```

Comment:
```suggestion
        /* eslint-disable */
        const verts = new Float32Array([
          -a, -a,  a, -a,
          -a,  a, -a,  a,
           a, -a,  a,  a,
         ]);
         /* eslint-enable */
```
That's more readable as the params stands for separate vertices.

Reply:
Good call!